docs/content/how-to/disaster-recovery/platform-guides/agent.md

```
@@ -120,12 +120,50 @@ When performing cross-cluster migration for the Agent platform, additional objec
 2. **Assisted Installer PostgreSQL database**: The Assisted Installer database must be backed up and restored on the destination cluster.
 3. **Include in backup manifest**: These resources can be included in the Velero Backup manifest to automate this process.
 
-!!! important
+## Pre-Backup Steps (Agent Only)
 
-    Cross-cluster migration for the Agent platform is not yet supported (no E2E coverage), but the procedure is documented here for reference.
+Before creating a backup for an Agent HostedCluster, the following steps are required to prevent the CAPI Agent provider from unbinding agents and destroying workloads during the backup process:
+
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)